src/pluggy/manager.py

```diff
@@ -25,6 +27,23 @@ def __init__(self, plugin, message):
         super(Exception, self).__init__(message)
 
 
+class DistFacade(object):
```

Comment:
I wonder if we should be sticking stuff like this in some kind of `_compat.py` module?
Not really pressing but I can see more components like this showing up as we move towards dropping py2?

Reply:
its a facade for a legacy api, it should eventually go altogether (note this is setuptools vs modern stuff)

Reply:
it's basically an adapter to make `importlib-metadata` look like `pkg-resources` -- it's not really a compatibility shim

Reply:
>to make importlib-metadata look like pkg-resources

Hehe, I always thought that's what a [compat shim](https://en.wikipedia.org/wiki/Shim_(computing)) *was*. Like I said it's no big deal I was just thinking about avoiding adapters being in core modules but it's a **really** minor quiff :wink:.